source/index.js

```diff
@@ -40,7 +40,7 @@ module.exports = async (input = 'patch', options) => {
 	options = {
 		cleanup: true,
 		tests: true,
-		publish: true,
+		shouldRunPublish: true,
```

Comment:
Definitely, but `publish` is not used as a variable anywhere else in the code base after the variable name change.

Reply:
Sure. Don't touch it if not necessary. :)

Reply:
Okay, but it’s an unused variable.

Reply:
Yes, in fact I think all these default values are useless, if no other entry for `index.js`. They have been set in `cli.js`. See https://github.com/sindresorhus/np/pull/497/files#diff-3268f296b25109ef2e796950ea09b998R83. But back to my words, don't touch it if not necessary, because `runPublish` doesn't need a default value as true.

Reply:
Okay. Reverted the change for this area.

Reply:
Ohhhhh I see now... these default values are defined for tests to pass! Once I removed the default (which would match the CLI defaults), the tests all failed because `runPublish` is false despite `publish` being true. I'll do a bit more refactoring so it's unnecessary.

Reply:
I'm trying to change as little as I can here... Let me know what you think of my solution. I think the better idea would be to define the same default blob as `options` in some sort of `beforeAll` or `beforeEach`, but I've never used this testing framework.

Reply:
Thanks for your investigation. Now some testing data in source file is strange. It's easy to know from ava's [API document](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#before--after-hooks). I can even accept you define `options` in the test file context and update each call of np with options in `test/index.js`.

```
- np('foo')
+ np('foo', options)
```

Reply:
Done https://github.com/sindresorhus/np/pull/497/commits/3eee2c80f4fb7c9ed27900c1585c63eca580161b